.github/workflows/build.yml

```diff
@@ -12,30 +12,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9"]
-        jupyterlab-version: [">=4.0,<5.0"]
+        python-version: ["3.14"]
+        jupyterlab-version: ["4.5"]
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Cache conda
-        uses: actions/cache@v4
-        env:
-          # Increase this value to reset cache
-          CACHE_NUMBER: 0
-        with:
-          path: ~/conda_pkgs_dir
-          key: ${{ runner.os }}-conda-${{ matrix.python-version }}-${{ env.CACHE_NUMBER }}-${{ hashFiles('pyproject.toml') }}
-          restore-keys: |
-            ${{ runner.os }}-conda-${{ matrix.python-version }}-${{ env.CACHE_NUMBER }}-${{ hashFiles('pyproject.toml') }}
-            ${{ runner.os }}-conda-${{ matrix.python-version }}
```

Comment:
Hey Konstantin, I just wanted to ask, is there a reason that this cache is removed for this job?